File: examples/widgets/msgbox/lv_example_msgbox_1.c
#include "../../lv_examples.h"
#if LV_USE_MSGBOX && LV_BUILD_EXAMPLES

static void event_cb(lv_event_t * e)
{
    lv_obj_t * btn = lv_event_get_target(e);
    lv_obj_t * label = lv_obj_get_child(btn, 0);
    LV_UNUSED(label);
    LV_LOG_USER("Button %s clicked", lv_label_get_text(label));
}

void lv_example_msgbox_1(void)
{
    lv_obj_t * mbox1 = lv_msgbox_create(NULL);

    lv_msgbox_add_title(mbox1, "Hello");

    lv_msgbox_add_text(mbox1, "This is a message box with two buttons.");
    lv_msgbox_add_close_button(mbox1);

    lv_obj_t * btn;
    btn = lv_msgbox_add_footer_button(mbox1, "Apply");
    lv_obj_add_event_cb(btn, event_cb, LV_EVENT_CLICKED, NULL);
    btn = lv_msgbox_add_footer_button(mbox1, "Cancel");
    lv_obj_add_event_cb(btn, event_cb, LV_EVENT_CLICKED, NULL);
    return;
}

#endif

File: examples/widgets/msgbox/lv_example_msgbox_1.py
def event_cb(e):
    btn = e.get_target_obj()
    label = btn.get_child(btn)
    print("Button %s clicked" % label.get_text())

mbox1 = lv.msgbox(lv.screen_active())
mbox1.add_title("Hello")
mbox1.add_text("This is a message box with two buttons")
mbox1.add_close_button()

btn = mbox1.add_footer_button("Apply")
btn.add_event_cb(event_cb, lv.EVENT.CLICKED, None)
btn = mbox1.add_footer_button("Cancel")
btn.add_event_cb(event_cb, lv.EVENT.CLICKED, None)
